python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -841,14 +841,16 @@ def get_backfills(
     ) -> Sequence[PartitionBackfill]:
         check.opt_inst_param(status, "status", BulkActionStatus)
         query = db_select([BulkActionsTable.c.body, BulkActionsTable.c.timestamp])
```

Comment:
I think we should go more aggressively.

We should deprecate `status` and error if they provide both `status` AND `filters`